CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ All notable changes to this project will be documented in this file.
 -   #1680 : Add support for `typing.Final`.
 -   Add a `--time_execution` flag to allow detailed investigation of critical sections of code.
 -   #1659 : Add multi-file support for classes.
+-   #1708 : Allow returning pointers from functions.
```

Comment:
No. We can only return pointers to arguments. Otherwise we would no longer be able to use `allocatable`. This would be a shame as there are many performance advantages associated with `allocatable`.

Clarified : da0d5875